Tests/SwiftDocCTests/Model/ParametersAndReturnValidatorTests.swift

```diff
@@ -569,7 +569,7 @@ class ParametersAndReturnValidatorTests: XCTestCase {
     }
     
     private let start = SymbolGraph.LineList.SourceRange.Position(line: 7, character: 6) // an arbitrary non-zero start position
-    private let symbolURL =  URL(fileURLWithPath: "/path/to/SomeFile.swift")
+    private let symbolURL = URL(fileURLWithPath: "/path/to/SomeFile.swift")
```

Comment:
Does this still need to be explicitly offset for these tests or can we use `makeSymbolGraph`'s defaults? I ask because this is very close to the default location values.

Reply:
My reason for this change is that by making the default values private variables, any test that wants to inspect and make assertions about these values would need to define customize the location and URL. If the test only needs a realistic value it can use the defaults. If the test has expectations about a specific value, it should define that value.

With this reasoning; it's intentional that this file uses a different URL than the default value. I'll update the default location as well so that it's not the same as this customized value.